modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -164,11 +165,11 @@ GLOBAL_LIST_EMPTY(cortical_borers)
 	///we dont want to spam the chat
 	var/deathgasp_once = FALSE
 	//the limit to the chemical and stat evolution
-	var/limited_borer = 10
+	var/limited_borer = 20
 	///borers can only enter biologicals if true
 	var/organic_restricted = TRUE
 	///borers are unable to enter changelings if true
-	var/changeling_restricted = TRUE
+	var/changeling_restricted = FALSE
```

Comment:
are you sure we want this? I vaguely remember people REALLY complaining about antag + antag.